server/option.go

```diff
@@ -290,9 +312,21 @@ func (o *scheduleOption) persist(kv *core.KV) error {
 
 func (o *scheduleOption) reload(kv *core.KV) error {
 	namespaces := make(map[string]NamespaceConfig)
-	for name, ns := range o.ns {
-		namespaces[name] = *ns.load()
+
+	f := func(k, v interface{}) bool {
+		var kstr string
+		var ok bool
+		if kstr, ok = k.(string); !ok {
+			return false
+		}
+		if ns, ok := v.(*namespaceOption); ok {
+			namespaces[kstr] = *ns.load()
+			return true
+		}
+		return false
 	}
+	o.ns.Range(f)
```

Comment:
em, this part is exactly the same with the code block in `persist`. How about extract a function?

Reply:
Sure!:ok_hand: